src/entity/Entity.ts

```diff
@@ -683,10 +683,10 @@ class Entity {
      * @param {Quat} rot - The new rotation quaternion.
      */
     public setRotation(rot: Quat) {
-        //this._qRot.copy(rot);
-        //
-        //@todo when necessary
-        //
+        this._pitchRad = rot.getPitch();
```

Comment:
Looks good, but feels like it should me more direct way. Will see. Now is ok, Just to remember.